src/toolkits/drawing_classifier/drawing_classifier.hpp

```diff
@@ -111,8 +116,8 @@ class EXPORT drawing_classifier: public ml_model_base {
       "    If dataset is an SFrame, it must have a column with the same name\n"
       "    as the feature column during training. Additional columns are\n"
       "    ignored.\n"
-      "    If the data is a single drawing, it can be either of type tc.Image,\n"
-      "    in which case it is a bitmap-based drawing input,\n"
+      "    If the data is a single drawing, it can be either of type "
+      "    tc.Image, in which case it is a bitmap-based drawing input,\n"
```

Comment:
Doesn't removing the newline make these two string literals concatenate into something way longer than the other lines here?